modules/data.atmosphere/R/download.NARR_site.R

```diff
@@ -73,10 +73,10 @@ download.NARR_site <- function(outfolder,
 
   narr_proc <- result_full %>%
     dplyr::mutate(
-      data_nc = purrr::map2(data, file, prepare_narr_year, lat = lat, lon = lon)
+      data_nc = purrr::map2(.data$data, file, prepare_narr_year, lat = lat, lon = lon)
```

Comment:
```suggestion
      data_nc = purrr::map2(.data$data, .data$file, prepare_narr_year, lat = lat, lon = lon)
```

Reply:
Hmm. @ashiklom is `narr_proc` needed at all? I don't see it used anywhere between here and the end of the function.